### ISSUE TITLE
upgrade to latest stable and compatible version of Mesos and Marathon

### ISSUE THREAD

**Issue by ryane** [CiscoCloud/microservices-infrastructure#863]
mesos 0.25.0?
marathon 0.11.1?

**Comment by siddharthist** [comment]
Does mantlui support Mesos 0.25? #739

**Comment by ryane** [comment]
it did seem to work in a brief test I did a couple of weeks ago. But, we do need to test it more thoroughly when we upgrade mesos

**Comment by Zogg** [comment]
Still needs confirmation, but calico seems to be working only with mesos from version 0.25 upwards.

**Comment by Zogg** [comment]
To be precise, calico works with freshly released 0.26.

**Comment by stevendborrelli** [comment]
I'm good with 0.26 and we should test Marathon 0.13+. 

We need to make sure the UI doesn't get broken after mesos updates. 

We can change the location of the UI during launch of the matters, and use assets from our own packages:
```
--webui_dir=VALUE	Directory path of the webui files/assets (default: /usr/local/share/mesos/webui)
```

**Comment by ryane** [comment]
fixed via #955
